deploy-as-code/helm/charts/core-services/egov-mdms-service/values.yaml

```diff
@@ -13,7 +13,7 @@ ingress:
 initContainers:
   gitSync:
     enabled: true
-    repo: "git@github.com:egovernments/egov-mdms-data"
+    repo: "git@github.com:misdwss/mdms-mgramseva"
```

Comment:
Raise a separate PR for this change. This isn't related to adapter master data.

Reply:
changed